build/run.js

```diff
@@ -146,6 +133,11 @@ commands.build.js = async function () {
     await run('npm', ['run', 'build'])
 }
 
+// We build WASM binaries from Rust code using `wasm-pack`. Intermediate temporary directory is used
+// before final copy to the Webpack's `dist` location because of two reasons:
+// 1. Webpack triggers recompilation on file changes, and we don't want to bother it until the final binaries are ready
+//   to use.
+// 2. `wasm-pack` clears its output directory before compilation, which breaks Webpack because of missing files.
```

Comment:
> 100 chars – check linter settings pls.

Reply:
I reduced the line length manually, but there is no config for `prettier` in our codebase. We also don't have a specific style guide for JS files, contributing guide only says to use `npx prettier` to format code, which I did. Let's address it in a separate task if we want to.

Reply:
I believe there is a config: https://github.com/enso-org/enso/blob/develop/.prettierrc.yaml
https://github.com/enso-org/enso/blob/develop/.prettierignore

Reply:
Oh, indeed. My editor doesn't show hidden files by default, so I missed that. It has `100` chars setting set though, I wonder if it is not applied to docstrings